dashboard/src2/pages/LoginSignup.vue

```diff
@@ -218,6 +249,12 @@ export default {
 		}
 	},
 	methods: {
+		isOauthLogin() {
+			return this.oauthEmailDomain === null;
```

Comment:
this function name is `isOauthLogin` but it returns `true` only if there is no oauth email domain?

Reply:
refactored to be more readable